frontend/src/common/constants/constants.ts

```diff
@@ -0,0 +1 @@
+export { DEFAULT_USER_ID } from 'jabber-shared/common/constants/constants';
```

Comment:
Move it to the user folder "frontend/src/common/constants/user" and then re-export from this file